### ISSUE TITLE
CSS: Manage accounts page

### ISSUE THREAD

**Issue by hannahkamundson** [codefordenver/ideaLab#330]
- [x] Change size of idealab logo to match Upload Page
- [x] Add page title under logo
	Manage Accounts
- [x] Spacing of top nav items for readability - **Emilie needs clarification: can someone provide padding recommendations? - ** thank you Patrick!
- [x] Change Color Availability click target is only on edges of the circle
- [x] Change Color Availability doesn't refill circle when you set it (only the inside border of the circle is colored)
- [x] Change Color Availability Checkmark/X are not very intuitive - not sure what to do here because I'm not sure what the functionality is
- [ ] Change Color Availability has no way to add another color - **Emilie needs clarification: I thought we decided to list all colors ourselves
- [x] Email Messages update titles to be more descriptive
	Message sent when status is changed to FAILED
	Message sent when status is changed to COMPLETED
- [x] Email Messages make textboxes larger
- [x] Email Messages align save button
- [x] Email Settings textboxes to match upload page
- [x] Dropbox Token Settings textbox to match upload page
- [x] Make save button drop shadows smaller
- [x] Make save buttons match other buttons
- [x] Unable to update role for user profiles
- [x] User Profiles - colors of infoContainer are inline styled
- [x] Make change password button match other buttons

**Comment by anotherpatrickcollins** [comment]
Upload Page logo styling
```
width: 50%;
margin: auto;
margin-top: 2rem;
border-style: none;
```

Textarea styling on upload page:
```
border: 2px solid black;
padding: 0.5rem;
margin-top: 1rem;
margin-bottom: 1rem;
margin-left: 0;
margin-right: 0;
```

Input styling on upload page:
```
width: 100%;
display: block;
height: 2em;
border-radius: 10px;
outline: none;
font-size: 1.5rem;
```

Save button styling (make text all caps):
```
background-color: #577ba1;
box-shadow: 2px 2px rgba(0,0,0,.5);
```

Password button styling (make text all caps):
```
border-radius: 3px;
```

**Comment by anotherpatrickcollins** [comment]
Idea for navbar spacing:
- Not sure how to move the underline ::after over
- 	Add to .statusMenu li {};
`padding-right: 1rem;`
- 	Add to .statusMenu button {}
`padding-left: 1rem;`

**Comment by anotherpatrickcollins** [comment]
Idea for infoContainer:

- make two classes for different rows
- make blue rows rgba(0,174,239,.8)
- make grey rows rgba(255,255,255,.6)

**Comment by anotherpatrickcollins** [comment]
Some fixes in #337 but not all. Crossing off any that I see are fixed.
